.github/workflows/tld-update.yml

```diff
@@ -5,10 +5,17 @@ on:
   schedule:
     # Run once a day at 15:00 UTC
     - cron:  '0 15 * * *'
+    
+permissions:
```

Comment:
For this workflow in specfic, it had no permission defined so, without specifying the top level and job level permissions, it was using write-all, which includes lots of other permissions.

If you mean the change made at the deploy-site.yml, where I just moved the write permission to be job scoped instead of workflow scoped. By now it really does not change anything to the permissions used in the workflow since there is only one job. Although it is yet recommended to declare top level permissions with minimal permissions because:

1.  if none is defined in the top level and a new job is added to the workflow later, it will have the write-all if no job level permission is defined.
2. if the top level has the write permissions a new job applied later will have the write permission even if it doesn't need it.

That's why, even in these cases, it is consider a good practice to define top level and job level permissions.

But it is indeed more verbose than simply applying the permissions to top level.